localstack/utils/coverage_docs.py

```diff
@@ -10,6 +10,10 @@ def get_coverage_link_for_service(service_name: str, action_name: str) -> str:
 
     available_services = SERVICE_PLUGINS.list_available()
 
+    # TODO remove this once the sqs-query API has been phased out
```

Comment:
https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-json-faqs.html#query-protocol

I don't think the SQS Query (XML) implementation is going to be phased out soon